pkg/cmd/marathon/app/app_kill.go

```diff
@@ -13,13 +17,62 @@ func newCmdMarathonAppKill(ctx api.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "kill",
 		Short: "Kill a running application instance.",
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if scale && host != "" {
+				return fmt.Errorf("the flags 'scale' and 'host' cannot be used at the same time")
+			}
+			return nil
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			return marathonAppKill(ctx, *client, args[0], scale, host)
 		},
 	}
 
-	cmd.Flags().BoolVar(&scale, "scale", false, "Scale the app down after performing the the operation.")
+	cmd.Flags().BoolVar(&scale, "scale", false, "Scale the app down after performing the operation.")
 	cmd.Flags().StringVar(&host, "host", "", "The hostname that is running app.")
 
 	return cmd
 }
+
+func marathonAppKill(ctx api.Context, client marathon.Client, appID string, scale bool, host string) error {
+	if scale {
+		_, err := client.API.Application(appID)
+		if apiErr, ok := err.(*goMarathon.APIError); ok && apiErr.ErrCode == goMarathon.ErrCodeNotFound {
```

Comment:
What if there's another error? Shouldn't we exit in those cases as well?

Reply:
:+1: @ArmandGrillet Can you add a test for this case?